### DISCUSSION TITLE
Uses a specific version rather than a nightly build

### DISCUSSION THREAD

**Discussion by llaville** [shivammathur/setup-php#646] [Q&A / Help] [answered]
Hello,

As explained in documentation (https://github.com/shivammathur/setup-php#nightly-build-setup), when we specify '8.2', it will use a nightly build also referenced as `8.2.0-dev` 

Due to my code that use a lot of `version_compare`, the development version is always lower than an `alpha`, `beta` or `RC`.

Is it possible to specify a full version like `8.2.0RC2` rather than just `8.2` ?

**Comment by shivammathur** [comment, marked as answer]
Currently, this is not supported as the builds are nightly from the branch, and specific alpha, beta, RC tags are not built.
